scipy/stats/_discrete_distns.py

```diff
@@ -6,6 +6,7 @@
 from scipy import special
 from scipy.special import entr, logsumexp, betaln, gammaln as gamln, zeta
 from scipy._lib._util import _lazywhere, rng_integers
+from scipy.interpolate import interp1d
```

Comment:
We already had a dependency on `interpolate` in `_continuous_distns.py`, so this is okay.